org.emoflon.ibex.tgg.ide/src/org/emoflon/ibex/tgg/ide/admin/IbexTGGBuilder.java

```diff
@@ -302,6 +302,20 @@ public static Resource loadModelInProject(IFile file, ResourceSet rs) throws IOE
 		resource.load(null);
```

Comment:
Consider passing in the typical options hash map, which can be null but doesn't have to be.  Hard coding this to null here limits the usefulness of this helper method.